rainier-core/src/main/scala/com/stripe/rainier/optimizer/Optimizer.scala

```diff
@@ -0,0 +1,25 @@
+package com.stripe.rainier.optimizer
+
+import com.stripe.rainier.sampler.DensityFunction
+
+object Optimizer {
+  def lbfgs(df: DensityFunction): Array[Double] = {
```

Comment:
Given how opaque the LBFGS library is, it might be worth a lot of comments in this method explaining how it's using the library.

Reply:
Will take as a follow-up.